bee-message/src/constants.rs

```diff
@@ -0,0 +1,19 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use core::ops::{Range, RangeInclusive};
+
+/// The total number of IOTA tokens in circulation.
+pub const IOTA_SUPPLY: u64 = 2_779_530_283_277_761;
```

Comment:
Should this constant really stay in the bee-message crate or maybe placed somewhere outside of this crate? Is it the job of this crate to define that constant?